src/components/TechTalkGroups/TechTalkGroups.js

```diff
@@ -43,37 +43,37 @@ const TechTalks = ({techTalks}) => {
   return (
     <Box m="auto" maxWidth={theme.breakpoints.values.lg}>
       <Typography variant="h1" color="textPrimary">
-        Tech Talks
+        Tech Talk Groups
       </Typography>
-      {techTalks
+      {techTalkGroups
         .slice((page - 1) * ITEMS_PER_PAGE, page * ITEMS_PER_PAGE)
```

Comment:
Should this be at done at the service layer level?

Reply:
Do you mean in this component's parent?